### ISSUE TITLE
Magepack does not exit when puppeteer times out

### ISSUE THREAD

**Issue by gwharton** [magesuite/magepack#111]
```
ℹ Collecting bundle modules in the browser.                                                                                                                                                                             18:47:31
ℹ Collecting modules for bundle "category".                                                                                                                                                                             18:47:31

 ERROR  Navigation timeout of 30000 ms exceeded                                                                                                                                                                         18:48:02

  at node_modules/puppeteer/lib/LifecycleWatcher.js:142:21
  -- ASYNC --
  at Frame.<anonymous> (node_modules/puppeteer/lib/helper.js:111:15)
  at Page.goto (node_modules/puppeteer/lib/Page.js:672:49)
  at Page.<anonymous> (node_modules/puppeteer/lib/helper.js:112:23)
  at Object.category (node_modules/magepack/lib/generate/collector/category.js:36:16)
  at processTicksAndRejections (internal/process/task_queues.js:97:5)
  at async module.exports (node_modules/magepack/lib/generate.js:24:13)

```
It hangs here and never returns to the command line. Can it, so that I can deal with the error?

**Comment by fredden** [comment]
This seems to have been fixed in #114.

**Comment by gwharton** [comment]
Perfect, didnt spot that. 

Cheers
